### PR TITLE
Ov 60 validation

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Ov 60 validation

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG file.

### CODE REVIEW

**Comment by VenkateshRavula** [comment]
Even though the file count seems huge, half of the file changes are dummy changes to make their timestamps recent. Please review

File: examples/appliance_device_snmp_v3_trap_destinations.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 options = {
     "type": "Destination",
     "destinationAddress": "1.1.1.1",
-    "userId": "a5c9a94d-ce98-42bd-8f3a-b1856d6071b0",
+    "userId": "",
```
Comment (yuvirani) [inline examples/appliance_device_snmp_v3_trap_destinations.py:33]: You can remove this assignment as you are assigning the value down

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,49 @@
...
 #### Features supported
 - Appliance Configuration Timeconfig
+- Appliance SNMPv1 Trap Destinations
 - Appliance SNMPv3 Trap Destinations
-- Appliance SNMPv3  Users
+- Appliance SNMPv3 Users
 - Appliance SSH Access
 - Appliance Time and Locale Configuration
+- Artifact Bundles
+- Connection Templates
```
Comment (yuvirani) [inline CHANGELOG.md:13]: Move below connection template

File: examples/appliance_device_snmp_v3_trap_destinations.py (hunk 1 above)
Comment (VenkateshRavula) [reply examples/appliance_device_snmp_v3_trap_destinations.py:33]: I have added empty userId key in the payload just to make people understand that "userId" is an important attribute in this payload.

File: CHANGELOG.md (hunk 2 above)
Comment (VenkateshRavula) [reply CHANGELOG.md]: done

**Review by yuvirani — APPROVED** [review]
LGTM

**Review by SHANDCRUZ — APPROVED** [review]

**Pull request merged**